vignettes/classify-cbd.Rmd

```diff
@@ -183,57 +183,81 @@ los_table_nas = cycle_net_los |>
 ```{r}
 #| eval: false
 #| echo: false
-los_table_long_missing = tibble::tribble(
+los_table_long_missing <- tibble::tribble(
   ~`Speed Limit (mph)`, ~AADT, ~cycle_segregation, ~level_of_service,
-  "30 mph", "4000+", "Painted Cycle Lane", 1,
-  "40 mph", "4000+", "Painted Cycle Lane", 1,
-  "40 mph", "4000+", "Painted Cycle Lane", 1,
-  "50 mph", "4000+", "Painted Cycle Lane", 0,
-  "60+ mph", "4000+", "Painted Cycle Lane", 0,
-  "40 mph", "2000 to 3999", "Painted Cycle Lane", 1,
-  "50 mph", "2000 to 3999", "Painted Cycle Lane", 0,
-  "60+ mph", "2000 to 3999", "Painted Cycle Lane", 0,
-  "40 mph", "0 to 1999", "Painted Cycle Lane", 1,
-  "50 mph", "0 to 1999", "Painted Cycle Lane", 0,
-  "60+ mph", "0 to 1999", "Painted Cycle Lane", 0,
-  "30 mph", "4000+", "Segregated Track (narrow)", 1,
-  "40 mph", "4000+", "Segregated Track (narrow)", 1,
-  "50 mph", "4000+", "Segregated Track (narrow)", 0,
-  "60+ mph", "4000+", "Segregated Track (narrow)", 0,
-  "40 mph", "2000 to 3999", "Segregated Track (narrow)", 1,
-  "50 mph", "2000 to 3999", "Segregated Track (narrow)", 0,
-  "60+ mph", "2000 to 3999", "Segregated Track (narrow)", 0,
-  "40 mph", "0 to 1999", "Segregated Track (narrow)", 1,
-  "50 mph", "0 to 1999", "Segregated Track (narrow)", 1,
-  "60+ mph", "0 to 1999", "Segregated Track (narrow)", 0,
-  "30 mph", "4000+", "Segregated Track (wide)", 2,
-  "30 mph", "2000 to 3999", "Segregated Track (wide)", 2,
-  "30 mph", "0 to 1999", "Segregated Track (wide)", 2,
-  "40 mph", "4000+", "Segregated Track (wide)", 1,
-  "40 mph", "2000 to 3999", "Segregated Track (wide)", 1,
-  "40 mph", "0 to 1999", "Segregated Track (wide)", 1,
-  "50 mph", "4000+", "Segregated Track (wide)", 1,
-  "50 mph", "2000 to 3999", "Segregated Track (wide)", 1,
-  "50 mph", "0 to 1999", "Segregated Track (wide)", 1,
-  "60+ mph", "4000+", "Segregated Track (wide)", 1,
-  "60+ mph", "2000 to 3999", "Segregated Track (wide)", 1,
-  "60+ mph", "0 to 1999", "Segregated Track (wide)", 1,
-  "20 mph",  "0 to 1999", "Mixed Traffic Street", 3,
-  "20 mph",  "2000 to 3999", "Mixed Traffic Street", 2,
-  "20 mph",  "4000+", "Mixed Traffic Street", 1,
-  "30 mph", "0 to 1999", "Mixed Traffic Street", 2,
-  "30 mph", "2000 to 3999", "Mixed Traffic Street", 1,
-  "30 mph", "4000+", "Mixed Traffic Street", 1,
-  "40 mph", "0 to 1999", "Mixed Traffic Street", 1,
-  "40 mph", "2000 to 3999", "Mixed Traffic Street", 0,
-  "40 mph", "4000+", "Mixed Traffic Street", 0,
-  "50 mph", "0 to 1999", "Mixed Traffic Street", 1,
-  "50 mph", "2000 to 3999", "Mixed Traffic Street", 0,
-  "50 mph", "4000+", "Mixed Traffic Street", 0,
-  "60+ mph", "0 to 1999", "Mixed Traffic Street", 0,
-  "60+ mph", "2000 to 3999", "Mixed Traffic Street", 0,
-  "60+ mph", "4000+", "Mixed Traffic Street", 0
+  # Painted Cycle Lane -> Cycle Lane
+  "30 mph", "4000+", "Painted Cycle Lane", 1, # CbD: 30-50kph, 4000+, Cycle Lane = ● (1)
```

Comment:
TBC: I think 30 mph should probably be in the 50 to 65 kph category, as per below:

```
20 mph -> 30 kph to 50 kph
30 mph -> 50 kph to 65 kph
40 mph -> 65 kph to 80 kph
50 mph -> 80 kph to 95 kph
60+mph -> 95 kph to 110 kph
```

Reasoning from email: 

> t's likely that more than 15% (the 85th percentile) of cars go faster than 20 mph on most 20 mph roads and that more than 15% of cars go more than 31 mph (50 kph) on most 30 mph roads.